scripts/Muon/GUI/Common/muon_workspace.py

```diff
@@ -0,0 +1,164 @@
+# pylint: disable=F0401
```

Comment:
we should change the name is it is not a muon workspace, so this might be confusing